SimCalorimetry/HcalTrigPrimAlgos/src/HcalTriggerPrimitiveAlgo.cc

```diff
@@ -553,6 +552,9 @@ void HcalTriggerPrimitiveAlgo::analyzeQIE11(IntegerCaloSamples& samples,
   IntegerCaloSamples output(samples.id(), tpSamples);
   output.setPresamples(tpPresamples);
 
+  // Based on the |ieta| of the sample, retrieve the correct region veto threshold
+  unsigned int theVetoThreshold = vetoThresholds_[theIeta];
```

Comment:
`theVetoThreshold` is copied from a integer here, and only used to be compared with integers in [L586](https://github.com/cms-sw/cmssw/pull/44373/files#diff-ba5256614636c69492dc581b4631046ea069da466ec01fd259a120e5d4526542R586) and [L591](https://github.com/cms-sw/cmssw/pull/44373/files#diff-ba5256614636c69492dc581b4631046ea069da466ec01fd259a120e5d4526542R591): why  do you define it as `unsigned`?

Reply:
On L591, the `sum` object is an `IntegerCaloSamples` object, which stores its information as `uint32_t`, so an `unsigned int` is constructed for `theVetoThreshold` to avoid compiler time comparison warnings.